docs/methods.rst

```diff
@@ -0,0 +1,47 @@
+Methods
+=======
+
+Several methods exist to help combine multiple data sets and convert between
+equivalent indices.
+
+ACE
+---
+
+Supports the ACE instrument by providing reference and acknowledgement
+information.
+
+
+.. automodule:: pysatSpaceWeather.instruments.methods.ace
```

Comment:
Methods show up on RTD for Dst but not for Ace, F10.7, nor the kp ap pair..

Reply:
It works flawlessly locally on this branch.

Reply:
It may be an issue with the pysat dependence, since it's grabbing pysat from pip.